### PR TITLE
include baseos contentTree as well before GC.

### PR DESCRIPTION
**Pull request by adarsh-zededa**
Issue: installDownloadedObject(<content-id>): InstallWorkResult error, exception while installing: error pulling <image> from containerd: image "<image>": not found

Reason: On boot, we only prevented ctrd images associated AppImg ContentTreeStatus from getting garbage collected and didn't include baseOs ContentTreeStatus, which led to BaseOS images getting GC.

Fix: On boot include BaseOS ContentTreeStatus as well which will prevent baseOs images from getting garbage collected.

Signed-off-by: adarsh-zededa <adarsh@zededa.com>

### CODE REVIEW

File: pkg/pillar/cmd/volumemgr/handlecontent.go [hunk 1]
```diff
@@ -148,16 +149,24 @@ func lookupContentTreeStatusAny(ctx *volumemgrContext, key string) *types.Conten
...
 }
 
-func getAllAppContentTreeStatus(ctx *volumemgrContext) map[string]*types.ContentTreeStatus {
-	log.Tracef("getAllAppContentTreeStatus")
-	pub := ctx.publication(types.ContentTreeStatus{}, types.AppImgObj)
-	contentIDAndContentTreeStatusIntf := pub.GetAll()
+func getAllContentTreeStatus(ctx *volumemgrContext) map[string]*types.ContentTreeStatus {
+	var (
+		pub                               pubsub.Publication
+		contentIDAndContentTreeStatusIntf map[string]interface{}
```
Comment (eriknordmark) [inline pkg/pillar/cmd/volumemgr/handlecontent.go:155]: Shouldn't this be a map[string]*types.ContentTreeStatus?

Oh, I see this is just the local result from GetAll. no need to declare that.
And it is easier to read if there are several similar sounding long variable names.
for key, item := range pub.GetAll() {
does the job and is much easier to read.

**Review by eriknordmark — APPROVED** [review]
I'd prefer less confusing variable names.

File: pkg/pillar/cmd/volumemgr/handlecontent.go (hunk 1 above)
Comment (adarsh-zededa) [reply pkg/pillar/cmd/volumemgr/handlecontent.go]: Changed.

**Review by eriknordmark — APPROVED** [review]
Thanks

**Comment by rvs** [comment]
LGTM but I'm also curious as to what @deitch thinks

**Pull request merged**